task_scheduler/tasks/main.py

```diff
@@ -65,12 +83,19 @@
 # config1 = ConfigDbTask(query={},db_connection=redis_client)
 # db_task = DbTask(priority=0, config=config1)
 
-# r = db_task.config.db_connection.insert(data, "DATA1")
-# print(r)
-# r = db_task.config.db_connection.get("DATA1")
-# print(r)
-# r = db_task.config.db_connection.delete("DATA1")
-# print(r)
+# print(db_task.config.db_connection.insert(data, "config-1"))
+
+# print(db_task.config.db_connection.insert(data1, "config-2"))
+
+# print(db_task.config.db_connection.insert(data2, "config-3"))
+
+# print(db_task.config.db_connection.insert(data3, "api-1"))
+
+# print(db_task.config.db_connection.get("config-*"))
+
+# print(db_task.config.db_connection.delete("config-*"))
+
+# print(db_task.config.db_connection.get("*"))
 
 # redis_client2 = RedisDbConnection(
 #                         db_name="1", 
```

Comment:
this file is only for testing, right?

Reply:
The exception handling is good @cristiang96 , taking favipcj's suggestion the `task_scheduler/tasks/main.py `should be moved to `tests/ `changing its name to something more specific.

Reply:
Yes, that file is only for testing. I moved that file to ```tests\testing_mongo_and_redis_connection.py```.

Reply:
Thanks!